README.md

```diff
@@ -8,20 +8,11 @@
 <p align="center">
   <a href="http://protocol.ai"><img src="https://img.shields.io/badge/made%20by-Protocol%20Labs-blue.svg?style=flat-square" /></a>
   <a href="http://libp2p.io/"><img src="https://img.shields.io/badge/project-libp2p-yellow.svg?style=flat-square" /></a>
+  <a href="https://godoc.org/github.com/libp2p/go-libp2p"><img src="https://godoc.org/github.com/libp2p/go-libp2p?status.svg" /></a>
   <a href="http://webchat.freenode.net/?channels=%23libp2p"><img src="https://img.shields.io/badge/freenode-%23libp2p-yellow.svg?style=flat-square" /></a>
```

Comment:
Is this one still active?

Reply:
@marten-seemann for what it is worth, I removed it on rust-libp2p https://github.com/libp2p/rust-libp2p/commit/f8744175622198a03e81b5f8109aafaf8f3aae38

Reply:
Removed in 4cd0db6fb96bd679b51eacf44ee8848435b1f9fe.